### DISCUSSION TITLE
Hold up to taunt and optionally activate mode

### DISCUSSION THREAD

**Discussion by LeoKids** [ikemen-engine/Ikemen-GO#1284] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

In Naruto:Ultimate Ninja series,You need to taunt by holding up and potentially give eligible characters their modes,This happens with every character but eligible characters (Ex:Sakura (PTS and TS),Shikamaru (PTS and TS),Kiba (PTS and TS),and Asuma (only in NUN4).) can receive a mode just by taunting and their stats are stronger,Their attack increases and their stats improve.

In Mugen,you just press Start or the button used to taunt to actually taunt,Up is jumping and not taunting. Not a good thing if you wanna make NUN4 on the Ikemen engine,but useful enough for traditional fighting games like Street Fighet and KOF that are not meant to be input simplified for newcomers.

### Describe the solution you'd like

https://www.youtube.com/watch?v=VxK6P6W_nn4

https://www.youtube.com/watch?v=HnqQnDq2PHg

https://www.youtube.com/watch?v=Lb8FAC2j1go

https://www.youtube.com/watch?v=7-moPghRqTY

### Search terms

NUN5

**Comment by potsmugen** [comment, marked as answer]
This is something that has to be implemented in each character's coding (or as a common file). The engine itself already allows doing this.
